src/app/compiler.js

```diff
@@ -201,6 +201,13 @@ function Compiler (editor, handleGithubCall) {
       jobs.push({sources: source});
       worker.postMessage({cmd: 'compile', job: jobs.length - 1, source: JSON.stringify(source), optimize: optimize});
     };
+    if (!url.startsWith('http') && !url.startsWith('file')) {
```

Comment:
Since we're limiting it for the `https` case and using a local file, can you add a comment explaining the same (and reasons, i.e. https doesn't allow that)?